api/tests/core/mails/transactional/users/online_event_reminder_test.py

```diff
@@ -0,0 +1,104 @@
+import datetime
+
+import pytest
+
+from pcapi.core.bookings import factories as bookings_factories
+from pcapi.core.categories import subcategories_v2 as subcategories
+import pcapi.core.mails.testing as mails_testing
+from pcapi.core.mails.transactional.users import online_event_reminder
+from pcapi.core.offers import factories as offers_factories
+
+
+pytestmark = pytest.mark.usefixtures("db_session")
+
+
+class OnlineEventReminderTest:
+    def test_get_online_events_happening_soon(self) -> None:
+        now = datetime.datetime.utcnow()
+        in_1_hour = now + datetime.timedelta(hours=1)
```

Comment:
Suggestion: pourquoi définir  in_1_houre er in 30_minutes
Je trouve qu'on peux directement faire 
in_50_minutes = now + datetime.timedelta(minutes=50)
...